lib/sedna/core/lifelong_learning/lifelong_learning.py

```diff
@@ -78,55 +90,102 @@ class LifelongLearning(JobBase):
-            task_definition=task_definition,
-            task_relationship_discovery=task_relationship_discovery,
-            task_mining=task_mining,
-            task_remodeling=task_remodeling,
-            inference_integrate=inference_integrate,
-            unseen_task_detect=unseen_task_detect
+            estimator,
+            task_definition=None,
+            task_relationship_discovery=None,
+            task_allocation=None,
+            task_remodeling=None,
+            inference_integrate=None,
+            task_update_decision=None,
+            unseen_task_allocation=None,
+            unseen_sample_recognition=None,
+            unseen_sample_re_recognition=None,
         )
     """
 
     def __init__(self,
                  estimator,
                  task_definition=None,
                  task_relationship_discovery=None,
-                 task_mining=None,
+                 task_allocation=None,
                  task_remodeling=None,
                  inference_integrate=None,
-                 unseen_task_detect=None):
-
-        if not task_definition:
-            task_definition = {
-                "method": "TaskDefinitionByDataAttr"
-            }
-        if not unseen_task_detect:
-            unseen_task_detect = {
-                "method": "TaskAttrFilter"
-            }
-        e = MulTaskLearning(
+                 task_update_decision=None,
+                 unseen_task_allocation=None,
+                 unseen_sample_recognition=None,
+                 unseen_sample_re_recognition=None
+                 ):
+
+        e = SeenTaskLearning(
             estimator=estimator,
             task_definition=task_definition,
             task_relationship_discovery=task_relationship_discovery,
-            task_mining=task_mining,
+            seen_task_allocation=task_allocation,
             task_remodeling=task_remodeling,
-            inference_integrate=inference_integrate)
-        self.unseen_task_detect = unseen_task_detect.get("method",
-                                                         "TaskAttrFilter")
-        self.unseen_task_detect_param = e._parse_param(
-            unseen_task_detect.get("param", {})
+            inference_integrate=inference_integrate
         )
+
+        self.unseen_sample_recognition = unseen_sample_recognition or {
+            "method": "SampleRegonitionDefault"
+        }
+        self.unseen_sample_recognition_param = e._parse_param(
+            self.unseen_sample_recognition.get("param", {}))
+
+        self.unseen_sample_re_recognition = unseen_sample_re_recognition or {
+            "method": "SampleReRegonitionDefault"
+        }
+        self.unseen_sample_re_recognition_param = e._parse_param(
+            self.unseen_sample_re_recognition.get("param", {}))
+
+        self.task_update_decision = task_update_decision or {
+            "method": "UpdateStrategyDefault"
+        }
+        self.task_update_decision_param = e._parse_param(
```

Comment:
if `task_update_decision` is a callable module instance, then there's no need to set its param by `_parse_param`.